### ISSUE TITLE
Stardew Valley Frozen Quest Marnie’s Request (Cave Carrot)

### ISSUE THREAD

**Issue by zufardhiyaulhaq** [Pathoschild/SMAPI#553]
This happen when im using SMAPI and doing Quest
this is my error log

> [SMAPI] An error occured in the base update loop: System.FormatException: Input string was not in a correct format.
>   at System.Number.ParseSingle (System.String value, NumberStyles options, System.Globalization.NumberFormatInfo numfmt) <0x40281460 + 0x0017b> in <filename unknown>:0                                                                                         
>   at System.Single.Parse (System.String s, NumberStyles style, System.Globalization.NumberFormatInfo info) <0x40281420 + 0x00017> in <filename unknown>:0                                                                                                       
>   at System.Single.Parse (System.String s, IFormatProvider provider) <0x4028e1a0 + 0x0002b> in <filename unknown>:0             
>   at System.Convert.ToSingle (System.String value) <0x4140d7c0 + 0x0002f> in <filename unknown>:0                               
>   at StardewValley.Event.checkForNextCommand (StardewValley.GameLocation location, Microsoft.Xna.Framework.GameTime time) <0x4131e000 + 0x07fcf> in <filename unknown>:0                                                                                        
>   at StardewValley.GameLocation.UpdateWhenCurrentLocation (Microsoft.Xna.Framework.GameTime time) <0x41286ce0 + 0x00ec3> in <filename unknown>:0                                                                                                                
>   at StardewValley.Game1.UpdateLocations (Microsoft.Xna.Framework.GameTime time) <0x41284b90 + 0x00383> in <filename unknown>:0 
>   at StardewValley.Game1.Update (Microsoft.Xna.Framework.GameTime gameTime) <0x40365ce0 + 0x030cb> in <filename unknown>:0      
>   at StardewModdingAPI.Framework.SGame.Update (Microsoft.Xna.Framework.GameTime gameTime) <0x402994c0 + 0x039eb> in <filename unknown>:0                                                                                                                        
> [SMAPI] An error occured in the base update loop: System.FormatException: Input string was not in a correct format.             
>   at System.Number.ParseSingle (System.String value, NumberStyles options, System.Globalization.NumberFormatInfo numfmt) <0x40281460 + 0x0017b> in <filename unknown>:0                                                                                         
>   at System.Single.Parse (System.String s, NumberStyles style, System.Globalization.NumberFormatInfo info) <0x40281420 + 0x00017> in <filename unknown>:0                                                                                                       
>   at System.Single.Parse (System.String s, IFormatProvider provider) <0x4028e1a0 + 0x0002b> in <filename unknown>:0             
>   at System.Convert.ToSingle (System.String value) <0x4140d7c0 + 0x0002f> in <filename unknown>:0                               
>   at StardewValley.Event.checkForNextCommand (StardewValley.GameLocation location, Microsoft.Xna.Framework.GameTime time) <0x4131e000 + 0x07fcf> in <filename unknown>:0                                                                                        
>   at StardewValley.GameLocation.UpdateWhenCurrentLocation (Microsoft.Xna.Framework.GameTime time) <0x41286ce0 + 0x00ec3> in <filename unknown>:0                                                                                                                
>   at StardewValley.Game1.UpdateLocations (Microsoft.Xna.Framework.GameTime time) <0x41284b90 + 0x00383> in <filename unknown>:0 
>   at StardewValley.Game1.Update (Microsoft.Xna.Framework.GameTime gameTime) <0x40365ce0 + 0x030cb> in <filename unknown>:0      
>   at StardewModdingAPI.Framework.SGame.Update (Microsoft.Xna.Framework.GameTime gameTime) <0x402994c0 + 0x039eb> in <filename unknown>:0                                                                                                                        
> [SMAPI] An error occured in the base update loop: System.FormatException: Input string was not in a correct format.             
>   at System.Number.ParseSingle (System.String value, NumberStyles options, System.Globalization.NumberFormatInfo numfmt) <0x40281460 + 0x0017b> in <filename unknown>:0                                                                                         
>   at System.Single.Parse (System.String s, NumberStyles style, System.Globalization.NumberFormatInfo info) <0x40281420 + 0x00017> in <filename unknown>:0                                                                                                       
>   at System.Single.Parse (System.String s, IFormatProvider provider) <0x4028e1a0 + 0x0002b> in <filename unknown>:0             
>   at System.Convert.ToSingle (System.String value) <0x4140d7c0 + 0x0002f> in <filename unknown>:0                               
>   at StardewValley.Event.checkForNextCommand (StardewValley.GameLocation location, Microsoft.Xna.Framework.GameTime time) <0x4131e000 + 0x07fcf> in <filename unknown>:0                                                                                        
>   at StardewValley.GameLocation.UpdateWhenCurrentLocation (Microsoft.Xna.Framework.GameTime time) <0x41286ce0 + 0x00ec3> in <filename unknown>:0                                                                                                                
>   at StardewValley.Game1.UpdateLocations (Microsoft.Xna.Framework.GameTime time) <0x41284b90 + 0x00383> in <filename unknown>:0 
>   at StardewValley.Game1.Update (Microsoft.Xna.Framework.GameTime gameTime) <0x40365ce0 + 0x030cb> in <filename unknown>:0      
>   at StardewModdingAPI.Framework.SGame.Update (Microsoft.Xna.Framework.GameTime gameTime) <0x402994c0 + 0x039eb> in <filename unknown>:0

**Comment by zufardhiyaulhaq** [comment]
![screenshot from 2018-06-24 15 34 09](https://user-images.githubusercontent.com/11990726/41817375-15f0a33a-77c4-11e8-8967-760286230ba6.png)

**Comment by vorpalhex** [comment]
I'm also hitting this error.
Smapi 2.6-beta.17 with Stardew Valley 1.3.21 on Win 10

```
[12:33:01 ERROR SMAPI] An error occured in the base update loop: System.FormatException: Input string was not in a correct format.
   at System.Number.ParseSingle(String value, NumberStyles options, NumberFormatInfo numfmt)
   at System.Convert.ToSingle(String value)
   at StardewValley.Event.checkForNextCommand(GameLocation location, GameTime time) in C:\Users\gitlab-runner\gitlab-runner\builds\5c0f9387\0\chucklefish\stardewvalley\Farmer\Farmer\Event.cs:line 1184
   at StardewValley.Event.checkForNextCommand(GameLocation location, GameTime time) in C:\Users\gitlab-runner\gitlab-runner\builds\5c0f9387\0\chucklefish\stardewvalley\Farmer\Farmer\Event.cs:line 787
   at StardewValley.GameLocation.UpdateWhenCurrentLocation(GameTime time) in C:\Users\gitlab-runner\gitlab-runner\builds\5c0f9387\0\chucklefish\stardewvalley\Farmer\Farmer\Locations\GameLocation.cs:line 1836
   at StardewValley.Game1.UpdateLocations(GameTime time) in C:\Users\gitlab-runner\gitlab-runner\builds\5c0f9387\0\chucklefish\stardewvalley\Farmer\Farmer\Game1.cs:line 3488
   at StardewValley.Game1.Update(GameTime gameTime) in C:\Users\gitlab-runner\gitlab-runner\builds\5c0f9387\0\chucklefish\stardewvalley\Farmer\Farmer\Game1.cs:line 2337
   at StardewModdingAPI.Framework.SGame.Update(GameTime gameTime) in C:\source\_Stardew\SMAPI\src\SMAPI\Framework\SGame.cs:line 665
```

**Comment by Pathoschild** [comment]
That's a bug in Stardew Valley 1.3.21. The developers are already aware of it, so it'll hopefully be fixed in 1.3.22. :)
